skycultures/arabic/description.ar.utf8

```diff
@@ -0,0 +1,283 @@
+<!DOCTYPE html>
+<html lang="ar" dir="rtl">
+<head>
+<title>العربي</title>
+</head>
```

Comment:
By formal rules no need header block for HTML'ed description, because we includes the descripting in our HTML page.